tutorial/.vscode/tasks.json

```diff
@@ -7,42 +7,42 @@
       "group": "build",
       "label": "Build All",
       "type": "shell",
-      "command": "docker run --rm -v${workspaceFolder}:/mnt/project dawidbuchwald/cc65-tools-make all",
+      "command": "make all",
```

Comment:
To me, this is the advantage, developing as if you had the tools installed locally.

Reply:
This only works with the extension installed. Should anyone decide to use pure docker-based build, it will not be possible anymore.